presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkConfig.java

```diff
@@ -383,14 +384,14 @@ public PrestoSparkConfig setMinHashPartitionCount(int minHashPartitionCount)
 
     public boolean isSparkResourceAllocationStrategyEnabled()
     {
-        return isResourceAllocationStrategyEnabled;
+        return resourceAllocationStrategyEnabled;
     }
 
     @Config("spark.resource-allocation-strategy-enabled")
     @ConfigDescription("Determines whether the resource allocation strategy for executor and partition count is enabled")
     public PrestoSparkConfig setSparkResourceAllocationStrategyEnabled(boolean isResourceAllocationStrategyEnabled)
     {
-        this.isResourceAllocationStrategyEnabled = isResourceAllocationStrategyEnabled;
+        this.resourceAllocationStrategyEnabled = isResourceAllocationStrategyEnabled;
```

Comment:
nit : `this.resourceAllocationStrategyEnabled = resourceAllocationStrategyEnabled`